tests/forms_tests/tests/test_fields.py

```diff
@@ -642,6 +643,23 @@ class DurationForm(Form):
             str(f['duration'])
         )
 
+    def test_durationfield_3(self):
+        """
+        DurationField.prepare_value functions properly when given a timedelta.
```

Comment:
I think you can combine the two tests and call it `test_durationfield_prepare_value()`. The test seems self explanatory so I don't think a docstring is needed.